bearer/bearer.go

```diff
@@ -268,6 +268,13 @@ func (b *Token) Sign(signer neofscrypto.Signer) error {
 	return nil
 }
 
+// SignedData returns actual payload for sign.
```

Comment:
`to sign`? I always remember @EliChin at these moments.

Reply:
updated